src/Raven.Client/Documents/Session/IAdvancedSessionOperations.Stream.cs

```diff
@@ -28,83 +28,77 @@ public partial interface IAdvancedSessionOperations :
     public interface IDocumentsSessionOperations
     {
         /// <summary>
-        ///     Stream the results on the query to the client, converting them to
-        ///     CLR types along the way.
-        ///     <para>Does NOT track the entities in the session, and will not includes changes there when SaveChanges() is called</para>
+        ///     Creates a stream to fetch documents that satisfy criteria <paramref name="query"/>.
+        /// <para>
+        ///     Documents are converted to CLR type <typeparamref name="T"/> along the way and wrapped in <see cref="StreamResult{T}"/> containing document metadata.
+        /// </para>
         /// </summary>
-        /// <param name="query">Query to stream results for</param>
+        /// <remarks>
+        ///     Does NOT track the entities in the session and will not include its changes when <see cref="IDocumentSession.SaveChanges"/> is called.
+        /// </remarks>
+        /// <param name="query">The criteria by which to fetch documents from the database</param>
+        /// <returns>An enumerator used to iterate over the collection of documents received via the stream</returns>
```

Comment:
We can inherit the comment from the below method, can't we? The one below has more parameters so it will be good. See the following comment https://github.com/ravendb/ravendb/pull/18142#discussion_r1512232230